blog/_posts/2020-03-04-jan-feb-2020-commit-of-the-month.md

```diff
@@ -0,0 +1,32 @@
+---
+slug: jan-feb-2020-commit-of-the-month
+date: 2020-03-04
+author: maia
+layout: blog
+title: "Commit of the Month(s): Docker Push & Pull Progress"
+subtitle: "It's 10PM; Do You Know Where YOUR Progress Meter Is?"
+image: /assets/images/jan-feb-2020-commit-of-the-month/featuredImage.jpg
+image_caption: "Photo by Dylan (newchaos) on <a href='https://www.flickr.com/photos/newchaos/411409012/in/photolist-CmzyQ-F61dsG-6F3spX-mbzo9P-4zUE6i-4sNtuF-2d5kfHa-gvc6SZ-8Jms3t-Pn8o5D-7qbih-fsMbTJ-7oi3xg-ekgkNn-8HbgUT-6v43xq-7JBbWG-haQasf-du5kXV-81eEex-7Jxg16-2zc92-5KF6Zw-5KASgB-7Mc9cm-akYWET-8CTya-k1bGyi-98qSsb-4B7y9i-4B7y9Z-8CTQZ-a12btq-8CTN3-5xUp5h-efQEZt-bpA9kC-s5hAvG-cdD8Bh-99hp5v-8CTvj-8CTMB-hAJuZS-8CTKw-8CTQu-7EHjNt-77vUt2-8CTEC-8CTrQ-8CTpu'>Flickr</a>.)"
+image_type: "contain"
+tags:
+  - docker
+  - tilt
+  - cotm
+keywords:
+  - tilt
+  - docker pull
+  - docker push
+  - transparency
+  - visibility
+---
+We missed our Commit of the Month post for January, so this post is doing double duty. We'll be discussing two exciting commits, which are two sides of the same coin: [e77be9fe](https://github.com/windmilleng/tilt/commit/e77be9fe2d97f5893ea99f131a1ef2ffdb4ec576) (Jan.) and [467f1913](https://github.com/windmilleng/tilt/commit/467f1913ea08b562e2e16b26d8d7e458b92c941b) (Feb.) **print `docker push` and `docker pull` progress (respectively) to the Web UI**, so you're not in the dark about where your development time is going.
```

Comment:
I don't think this is accurate?

Here's the commit of the month post for January
https://blog.tilt.dev/2020/02/04/are-you-my-kustomize.html

Reply:
ha whoops, I missed it b/c it didn't have "cotm" in the title. Will fix.